dcos-log/journal/reader/config.go

```diff
@@ -44,10 +47,19 @@ func OptionMatch(m []JournalEntryMatch) Option {
 			return ErrUninitializedReader
 		}
 
-		for _, match := range m {
-			r.Journal.AddMatch(match.String())
+		fn := func(journal *sdjournal.Journal) {
+			for _, match := range m {
+				journal.AddMatch(match.String())
+				logrus.Infof("adding AND match %s", match)
```

Comment:
Does this need to be a logrus.Debugf?  If not, perhaps we could add more context.  Also, this comment applies to the disjunction matches below.